projects/EWTeam/build.xml

```diff
@@ -72,11 +78,7 @@ No need for Javadoc - this is not an API.
         <copy file="${pc2.jar}" tofile="${lib.dir}/pc2.jar" overwrite="true"/>
     </target>
 
-    <target name="properties" depends="copy-pc2jar">
-		<available classname="org.eclipse.jdt.core.JDTCompilerAdapter" property="build.compiler" value="org.eclipse.jdt.core.JDTCompilerAdapter" />
-         <condition property="git_hash" value="${env.GIT_COMMIT}" else="HEAD">
-            <isset property="env.GIT_COMMIT"/>
-        </condition>
+    <target name="-execgit" unless="exists.GITHUB_RUN_NUMBER">
```

Comment:
I really do not understand this change.  On the other hand, it's only related to EWTeam, which I think we probably should just remove from the system, so I really don't care.  On the third hand, as a general rule it would be nice if changes were accompanied by some YAML comments explaining them...